superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -208,6 +211,28 @@ const ChangeDatasourceModal: FunctionComponent<ChangeDatasourceModalProps> = ({
     return data;
   };
 
+  const customGotoPage = (p: number) => {
+    SupersetClient.get({
```

Comment:
nit: use `makeApi` instead

https://github.com/apache/superset/blob/252c64b3979f15789f3f56c96ce896fa6cff8164/superset-frontend/src/SqlLab/components/ResultSet.tsx#L364

Reply:
even nit-ier nit... is `fetchPage` clearer than `customGotPage`?